unic/ucd/name/src/name.rs

```diff
@@ -11,16 +11,41 @@
 use core::cmp::Ordering;
 use core::fmt;
 
+extern crate unic_ucd_normal;
```

Comment:
I believe `extern crate` is expected to live in `lib.rs`.

Reply:
Fixed!